infrastructure/helm-chart/charts/core/templates/configmap.yaml

```diff
@@ -4,15 +4,8 @@ metadata:
   name: hostnames
   namespace: {{ .Values.global.kubernetes.namespace }}
 data:
-  {{ if .Values.global.ingress }}
-   {{ if .Values.global.ingress.https }}
-  HOST: https://{{ .Values.global.kubernetes.host }}
-   {{ else }}
-  HOST: http://{{ .Values.global.kubernetes.host }}
-   {{ end }}
-  {{ else }}
-  HOST: http://{{ .Values.global.kubernetes.host }}
-  {{ end }}
+  HOST: {{ .Values.defaultApiHost }}
```

Comment:
As `host` is no longer part of the `kubernetes section` - we have a default version for the apiHost, which can later be overwritten by the ingress. The ingress controller is responsible to route the traffic from outside of the cluster, so he should also have the knowledge on which url the API is accessible.